standard-dev.cfg

```diff
@@ -0,0 +1,45 @@
+# Standard development config to be used in standardized policies.
+# May or may not be used by more sophisticated policies.
+
+# Usage:
+# [buildout]
+# extends =
+#     test-policy.cfg
+#     https://raw.githubusercontent.com/4teamwork/gever-buidlouts/master/standard-dev.cfg
+#
+# ogds-db-name = ???
+
+[buildout]
+extends =
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/plone-development.cfg
+    http://buildout-proxy.4teamwork.ch/4teamwork/opengever-buildouts/master/ruby-gems.cfg
```

Comment:
👍 resolved be requiring the `development.cfg` in the policy to also extend `plone-development.cfg` in order to have less extension depth.. 😯